upload.sh

```diff
@@ -2,7 +2,24 @@
 
 set +x # Do not leak information
 
-RELEASE_NAME="continuous" # Do not use "latest" as it is reserved by GitHub
+# did the user request that this no replace their latest CI binary (called
+# contuous) - for example because this is a feature test branch?
+# or is this a tag (and therefore a release)?
+if [ ! -z $UPLOADTOOL_SUFFIX ] ; then
```

Comment:
Where is `$UPLOADTOOL_SUFFIX` set?

Reply:
I guess you can set it in the environment before you call uploadtool.

Reply:
That's what you set in your .travis.yml before calling the upload.sh
This way what I do doesn't change the default behavior

Reply:
Ah, OK. I's been some time... ;-)